beta/src/components/Layout/Sidebar/Sidebar.tsx

```diff
@@ -63,11 +63,11 @@ export function Sidebar({isMobileOnly}: {isMobileOnly?: boolean}) {
     <aside
       className={cn(
         `lg:flex-grow lg:flex flex-col w-full pt-4 pb-8 lg:pb-0 lg:max-w-xs fixed lg:sticky bg-wash dark:bg-wash-dark z-10`,
-        isOpen ? 'block z-40' : 'hidden lg:block'
+        isOpen ? 'block z-40' : 'hidden lg:block top-0'
       )}
       aria-hidden={isHidden ? 'true' : 'false'}
+      // seems like isHidden is undefined always
```

Comment:
If it is, remove the logic/prop?

Reply:
We have code logic to compute it. 
Do we have any plans of using it for later use? 
Just wanted to make sure if we need this or not.
https://github.com/reactjs/reactjs.org/blob/d7cbb1bab155f28519e86ddfff453896cb359568/beta/src/components/Layout/Sidebar/Sidebar.tsx#L21

Reply:
It's probably remaining from some old design. If it's not used we can remove.

Reply:
Fixed this.